lib/components/TestModal.js

```diff
@@ -0,0 +1,37 @@
+import React, { PureComponent } from 'react';
+import { Modal, Button, Text } from '@bpanel/bpanel-ui';
+
+class TestModal extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+    }
+  }
+
+  onClick() {
+    const { showModal } = this.state;
+    console.log('closemodal!');
```

Comment:
did you mean to leave this logger statement?

Reply:
nope! Good catch.